cli/internal/fs/turbo_json.go

```diff
@@ -347,3 +347,15 @@ func (c *TurboJSON) UnmarshalJSON(data []byte) error {
 
 	return nil
 }
+
+// MarshalJSON converts a TurboJSON into the equivalent json object in bytes
+// note: we go via rawTurboJSON so that the output format is correct
+func (c *TurboJSON) MarshalJSON() ([]byte, error) {
```

Comment:
If we Marshal the TurboJSON directly, the naming on the fields is incorrect (PascalCase)